test/utils/colors.test.js

```diff
@@ -0,0 +1,95 @@
+import { getContrastColorFrom, hexToRgb, hsvToRgb, rgbToHex, rgbToHsv } from '../../src/utils/colors';
+
+const Rgb_White = [255, 255, 255];
+const Rgb_Red = [255, 0, 0];
+const Hsv_Red = [0, 1, 1];
+const Rgb_Green = [0, 255, 0];
+const Hsv_Green = [120, 1, 1];
+const Rgb_Blue = [0, 0, 255];
+const Hsv_Blue = [240, 1, 1];
+const Rgb_Cyan = [0, 255, 255];
+const Hsv_Cyan = [180, 1, 1];
+const Rgb_Yellow = [255, 255, 0];
+const Hsv_Yellow = [60, 1, 1];
+const Rgb_Magenta = [255, 0, 255];
+const Hsv_Magenta = [300, 1, 1];
+const Rgb_Black = [0, 0, 0];
+
+describe('rgbToHex', () => {
+	it('should convert a rgb-array to hex-representation', () => {
+		expect(rgbToHex(null)).toBeNull();
+		expect(rgbToHex('foo')).toBeNull();
+		expect(rgbToHex([-1, -1, -1])).toBeNull();
+		expect(rgbToHex([0, 0, 0])).toBe('#000000');
+		expect(rgbToHex([186, 218, 85])).toBe('#bada55');
+		expect(rgbToHex([255, 255, 255])).toBe('#ffffff');
+		expect(rgbToHex([256, 256, 256])).toBeNull();
+	});
+});
+
+describe('hexToRgb', () => {
+	it('should convert a color hex-representation to a rgb-array', () => {
+		expect(hexToRgb()).toBeNull();
+		expect(hexToRgb(undefined)).toBeNull();
```

Comment:
`expect(hexToRgb()).toBeNull();` and expect(hexToRgb(undefined)).toBeNull();` tests both on undefined as parameter (language spec)